### PR TITLE
docs: add more content and translations

### PR DESCRIPTION
**Pull request by AnjaMis**
This PR adds more content regarding agility, kanban and rituals. Basically it adds content about our DOR, newly introduced rituals (backlog refinement, planning poker) and their translations.

🚨 Please check the french translation, I have no idea if what I added here makes sense or not. 

To be able to check translations in the localhost, run `yarn build` and then `yarn run serve`

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/current/process/kanban.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## La Definition of Ready ✅
+
+Une user story doit répondre à certains critères pour être prête à être développée. Une user story qui est prête à être développée à l'OKP4 doit avoir :
+
+- titre, description et déclaration de valeur
+- critères d'acceptation (scénarios)
+- maquettes (pour tous les appareils et thèmes)
+- ajout des tags
+- US présenté à l'équipe DEV (réunion d'amélioration)
+- attaché à une epic
```
Comment (MdechampG) [inline i18n/fr/docusaurus-plugin-content-docs/current/process/kanban.md:59]:
```suggestion
- rattachement à une epic
```

File: i18n/fr/docusaurus-plugin-content-docs/current/process/kanban.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- estimation (réunion de planification)
```
Comment (MdechampG) [inline i18n/fr/docusaurus-plugin-content-docs/current/process/kanban.md:60]:
```suggestion
- estimation (réunion de planning poker)
```

File: i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md [hunk 3]
```diff
@@ -1,34 +1,45 @@
...
-sidebar_position: 2
+sidebar_position: 3
 sidebar_label: "Nos rituels"
 hide_table_of_contents: true
 ---
 
 # Nos rituels
 
-📆 Réunions quotidiennes
+📆 **Réunions quotidiennes**
```
Comment (MdechampG) [inline i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md:9]:
```suggestion
📆 **Réunion quotidienne**
```

File: i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md [hunk 4]
```diff
@@ -1,34 +1,45 @@
...
 
 - Équipe technique
 - Utilisation et mise à jour régulière de notre tableau Kanban
 - Suivi du WIP
 
-📆 Réunions hebdomadaires
+📆 **Réunions hebdomadaires**
```
Comment (MdechampG) [inline i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md:15]:
```suggestion
📆 **Réunion hebdomadaire**
```

File: i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md [hunk 5]
```diff
@@ -1,34 +1,45 @@
...
 - Partage et célébration de nos progrès
 
-📆 Rétrospective
+📆 **Rétrospective**
 
 - Toutes les équipes
 - Améliorer nos processus et nos méthodes de travail
 
-📆 Réunions de projets
+📆 **Réunions de projets**
```
Comment (MdechampG) [inline i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md:31]:
```suggestion
📆 **Réunion de projets**
```

File: i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md [hunk 6]
```diff
@@ -1,34 +1,45 @@
...
 
 - Équipes de projet et de marketing et communication
```
Comment (MdechampG) [inline i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md:33]:
```suggestion
- Équipes projets et marketing et communication
```

File: i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md [hunk 7]
```diff
@@ -1,34 +1,45 @@
...
 - Examen de tous les projets en cours et à venir
+
+📆 **Le backlog refinement**
```
Comment (MdechampG) [inline i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md:36]:
```suggestion
📆 **Le raffinage du backlog**
```

File: i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md [hunk 8]
```diff
@@ -1,34 +1,45 @@
...
+
+- Réunion hebdomadaire entre l'équipe produit et l'équipe technique
+- L'équipe produit présente de nouvelles user story à l'équipe technique.
```
Comment (MdechampG) [inline i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md:39]:
```suggestion
- L'équipe produit présente de nouvelles user stories à l'équipe technique.
```

File: i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md [hunk 9]
```diff
@@ -1,34 +1,45 @@
...
+- Les tâches sont affinées en fonction du retour d'information
+
+📆 **Le planning poker**
+
+- Réunion hebdomadaire entre l'équipe produit et l'équipe technique
+- Pour estimer l'effort de travail pour les nouvelles user story
```
Comment (MdechampG) [inline i18n/fr/docusaurus-plugin-content-docs/current/process/rituals.md:45]:
```suggestion
- Pour estimer l'effort de travail pour les nouvelles user stories
```

**Review by MdechampG — CHANGES_REQUESTED** [review]

**Comment by MdechampG** [comment]
Thank you @AnjaMis only few things to correct :)

**Review by ccamel — APPROVED** [review]
Great 👍

**Review by MdechampG — APPROVED** [review]
LGTM Thanks @AnjaMis !

**Pull request merged**